### PR TITLE
NRI: use control switch configmap to enable hugepage injection

### PR DESCRIPTION
**Pull request by zshi-redhat**
Network Resource Injector (NRI) has switches to config map to enable
optional features such as enableHugePageDownApi,
enableHonorExistingResources etc. This commit allows Operator to
create controlSwitch configmap for NRI and removes injectHugepageDownApi
from webhook-server container arguments.

### CODE REVIEW

File: bindata/manifests/webhook/004-configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
```
Comment (adrianchiris) [inline bindata/manifests/webhook/004-configmap.yaml:1]: a comment on the name of the file.

should it be `004-configmaps.yaml` ?  or `004-nri-control-switches.yaml` ?

there may be other configmap that need to be deployed in the future from nri.

Comment (zshi-redhat) [reply bindata/manifests/webhook/004-configmap.yaml:1]: @adrianchiris I think it shall be `004-configmap.yaml`. we have other manifests that are named after the singular kubernetes resource such as rbac, service or webhook etc. Example like webhook contains both mutating and validating webhooks, but the name is singular.

Comment (adrianchiris) [reply bindata/manifests/webhook/004-configmap.yaml:1]: ack, makes sense. additional configmaps would be added to this file.

**Review by adrianchiris — APPROVED** [review]

**Pull request merged**